scripts/commands.js

```diff
@@ -2,8 +2,10 @@ require('dotenv').config()
 const { BOT_PREFIX } = process.env
 const Discord = require('discord.js')
 
+const changeEnv = require('../server.js')
+
 exports.study = async (message, args, guildRoles) => {
-    if(!typeof([args][0], 'int') && args) return 0
+    if(!typeof([args][0], 'int') && args) return message.reply(`To enter study mode:\n${BOT_PREFIX}study`)
     args
     ?
```

Comment:
Here, you use a one-line if-statement over multiple lines, which makes it more confusing, use a traditional if-statement and it would actually take less lines